pkg/bpf/bpf.go

```diff
@@ -204,21 +204,33 @@ func (l *BpfLoader) Stop() {
 
 func NewVersionMap(config *options.BpfConfig) *ebpf.Map {
 	var versionPath string
+	var versionMap *ebpf.Map
 	if config.AdsEnabled() {
-		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh/map/")
+		versionPath = filepath.Join(config.BpfFsPath + "/bpf_kmesh/map/kmesh_version")
```

Comment:
since you are touching these hardcoded string, i would suggest defining some constants for them